conf/config.json

```diff
@@ -1,15 +1,15 @@
 {
     "zookeeper": {
-        "connectionString": "127.0.0.1:2181/backbeat",
-        "autoCreateNamespace": false
+        "connectionString": "127.0.0.1:2181",
+        "autoCreateNamespace": true
     },
     "kafka": {
         "hosts": "127.0.0.1:9092"
     },
     "queuePopulator": {
         "cronRule": "*/5 * * * * *",
         "batchMaxRead": 10000,
-        "zookeeperPath": "/queue-populator",
+        "zookeeperPath": "/replication-populator",
```

Comment:
the populator can be used for other purposes than replication like lifecycle, hence may be better to leave it as is.

Reply:
oh good point, I can adjust the readme to use this name then.